Model.php

```diff
@@ -360,13 +360,13 @@ public function setSiteIds($idAlert, $idSites)
 
     public function deleteTriggeredAlertsForSite($idSite)
     {
-        $this->getDb()->query("DELETE FROM " . Common::prefixTable("alert_triggered") . " WHERE idsite = ?", $idSite);
+        $this->getDb()->query("DELETE FROM " . Common::prefixTable("alert_triggered") . " WHERE idsite = ?", array($idSite));
```

Comment:
personally I'd prefer to use the short array syntax `[]`

Reply:
Yeah ok I change this. I prefer too.

Reply:
@sgiehl Do you want i rename all array() to [] ?

Reply:
only if you change something in that line. Otherwise that would end up in too many changes and potential merge problems